processor/formatters_test.go

```diff
@@ -30,6 +30,18 @@ func TestPrintError(t *testing.T) {
 	printError("Testing print error")
 }
 
+func TestPrintWarnF(t *testing.T) {
+	printWarnf("Testing print error")
+}
+
+func TestPrintDebugF(t *testing.T) {
+	printDebugf("Testing print error")
+}
+
+func TestPrintTraceF(t *testing.T) {
+	printTracef("Testing print error")
+}
+
```

Comment:
I reserve the right to point this out next time you say that you're not that pedantic about test coverage ;)

Reply:
:P I thought it was an easy gain :) and since its displayed on the main page...